workspaces/spectacle/src/spectacle.ts

```diff
@@ -323,15 +323,7 @@ export async function makeSpectacle(opticContext: IOpticContext) {
         endpointNode: endpoints.EndpointNodeWrapper;
         requestNode: endpoints.RequestNodeWrapper;
       }) => {
-        let path = parent.endpointNode.path();
-        let parentPath = path.parentPath();
-        const components = [path.value];
-        while (parentPath !== null) {
-          components.push(parentPath.value);
-          path = parentPath;
-          parentPath = path.parentPath();
-        }
-        return Promise.resolve(components.reverse());
+        return Promise.resolve(parent.endpointNode.path().components());
```

Comment:
Major difference between `path().components()` is this creates `path.value`. I've updated the `PathComponent` resolvers to use the Wrapper class